compiler/rustc_parse/src/parser/item.rs

```diff
@@ -1688,6 +1689,9 @@ impl<'a> Parser<'a> {
         Ok((class_name, ItemKind::Union(vdata, generics)))
     }
 
+    /// This function parses the fields of record structs:
+    ///   - struct S { ... }
+    ///   - enum E { variant { ... } }
```

Comment:
```suggestion
    ///
    ///   - `struct S { ... }`
    ///   - `enum E { Variant { ... } }`
```

Pascal case for enum variants & backticks.